backend/app/routers/auth.py

```
@@ -85,27 +99,37 @@
     reset_link = f"http://localhost:8000/reset-password?token={reset_token}"
 
+
     # Use background tasks to send the email asynchronously
     background_tasks.add_task(send_reset_email, user.email, reset_link)
 
     return {"message": PASSWORD_RESET_EMAIL_SENT}
 
 
 @router.post("/reset-password")
+async def reset_password(
+    request_data: ResetPasswordRequest, db: Session = Depends(get_db)
```

Ruff (0.8.2)

Trailing comma missing

(COM812)